src/app/activity/activity.component.html

```diff
@@ -35,11 +35,13 @@ <h1>{{ activity.name }}</h1>
             <ion-icon name="list-box" slot="start" color="primary"></ion-icon>
           </ng-container>
           <ng-container *ngSwitchCase="'Assessment'">
-            <ion-icon name="clipboard" slot="start" color="primary"></ion-icon>
+            <ion-icon name="clipboard" slot="start" [class]="(task.isDueToday && task.status != 'done') ? 'pulsing' : 'ng-star-inserted ion-color ion-color-primary hydrated'" color="primary"></ion-icon>
```

Comment:
`task.status != 'done'` is not enough, it is the same with below `(task.status === '' || task.status === 'in progress')`